pyproject.toml

```diff
@@ -38,11 +36,6 @@ pytest-operator = "^0.37.0"
 pytest-operator-cache = {git = "https://github.com/canonical/data-platform-workflows", tag = "v22.0.0", subdirectory = "python/pytest_plugins/pytest_operator_cache"}
 pytest-operator-groups = {git = "https://github.com/canonical/data-platform-workflows", tag = "v22.0.0", subdirectory = "python/pytest_plugins/pytest_operator_groups"}
 juju = "^3.5.2.0"
-mysql-connector-python = "^8.4.0"
-psycopg2-binary = "^2.9.9"
-pymongo = "^4.9.1"
-kazoo = "^2.10.0"
-kafka-python = "^2.0.2"
```

Comment:
Not used by the tests, but by the test charm.

Reply:
CC: @deusebio JFYI only.